docs/source/API/reference.rst

```diff
@@ -60,10 +60,7 @@ Query
 .. autoclass:: asreview.models.query.ClusterQuery
    :members:
    :inherited-members:
-.. autoclass:: asreview.models.query.MaxRandomQuery
-   :members:
-   :inherited-members:
-.. autoclass:: asreview.models.query.MaxUncertaintyQuery
+.. autoclass:: asreview.models.query.MixedQuery
```

Comment:
Can you explain this?

Reply:
Yup. Shouldve been a separate PR hahaha. MaxRandomQuery and MaxUncertaintyQuery don't exist, and MixedQuery does. Probably something left over from a previous change.

Reply:
Yup, besides that, I think you are wrong. MaxRandomQuery and MaxUncertaintyQuery are new. https://github.com/asreview/asreview/blob/6e245a6e0ba8ab3bc249c5706114d8a360822912/asreview/models/query/__init__.py#L16-L18

I think your fork is not up to date :) Please revert these changes and then sync your fork in upcoming PRs.

Reply:
Or maybe you are rendering the docs with the current ASReview version and not with the development version. Because these models will be introduced for the first time in the upcoming release.

Reply:
That was exactly what was happening. Thank you.